### DISCUSSION TITLE
Website help page has some layout issues

### DISCUSSION THREAD

**Discussion by Mrered** [community/community#57138] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

[键盘快捷方式 - GitHub 文档](https://docs.github.com/zh/get-started/using-github/keyboard-shortcuts#source-code-editing)

Seems like there is a problem with the markdown table formatting🤪

<img width="811" alt="image" src="https://github.com/community/community/assets/34948506/405f54e3-f244-49d7-8bb6-c74a40729793">

**Comment by adamlui** [comment]
@Mrered How did you create this screenshot with the shadow border and rounded corners?

**Reply by anxumalo** [reply]
@Mrered Good spot! 

It looks like all the localised pages are affected, except Russian(?) which is mostly a copy of the English page.

**Reply by Mrered** [reply]
> @Mrered How did you create this screenshot with the shadow border and rounded corners?

I used an app called [iShot-ScreenShot Recording OCR on the Mac App Store (apple.com)](https://apps.apple.com/app/ishot-%E4%BC%98%E7%A7%80%E7%9A%84%E6%88%AA%E5%9B%BE%E8%B4%B4%E5%9B%BE%E5%BD%95%E5%B1%8F%E5%BD%95%E9%9F%B3ocr%E7%BF%BB%E8%AF%91%E5%8F%96%E8%89%B2%E6%A0%87%E6%B3%A8%E5%B7%A5%E5%85%B7/id1485844094?mt=12), and I think people would like beautiful screenshots. Since there is such a tool now, why don't I use it to let others see more beautiful screenshots?🥳

This is a free software, but it also has a one-time payment version [iShot Pro-ScreenShot Recording on the Mac App Store (apple.com)](https://apps.apple.com/app/ishot-pro-%E4%B8%93%E4%B8%9A%E7%9A%84%E6%88%AA%E5%9B%BE%E8%B4%B4%E5%9B%BE%E5%BD%95%E5%B1%8F%E5%BD%95%E9%9F%B3ocr%E7%BF%BB%E8%AF%91%E5%8F%96%E8%89%B2%E5%B7%A5%E5%85%B7/id1611347086?mt=12)🤩

There may be better software, but this software already meets my daily needs, and the price is quite reasonable for me.😜

**Reply by Mrered** [reply]
> @Mrered Good spot!
> 
> It looks like all the localised pages are affected, except Russian(?) which is mostly a copy of the English page.

Idk, but I can get the key informations in that chaos.🤪
